airbyte-integrations/connectors/source-redshift/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/RedshiftJdbcSourceAcceptanceTest.java

```diff
@@ -5,38 +5,35 @@
 package io.airbyte.integrations.io.airbyte.integration_tests.sources;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import io.airbyte.cdk.integrations.source.jdbc.AbstractJdbcSource;
 import io.airbyte.cdk.integrations.source.jdbc.test.JdbcSourceAcceptanceTest;
 import io.airbyte.commons.io.IOs;
 import io.airbyte.commons.json.Jsons;
 import io.airbyte.integrations.source.redshift.RedshiftSource;
 import java.nio.file.Path;
-import java.sql.JDBCType;
-import java.sql.SQLException;
-import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
 
 // Run as part of integration tests, instead of unit tests, because there is no test container for
 // Redshift.
-class RedshiftJdbcSourceAcceptanceTest extends JdbcSourceAcceptanceTest {
+class RedshiftJdbcSourceAcceptanceTest extends JdbcSourceAcceptanceTest<RedshiftSource, RedshiftTestDatabase> {
 
-  private JsonNode config;
-
-  private static JsonNode getStaticConfig() {
-    return Jsons.deserialize(IOs.readFile(Path.of("secrets/config.json")));
-  }
+  private static JsonNode config;
 
   @BeforeAll
   static void init() {
+    config = Jsons.deserialize(IOs.readFile(Path.of("secrets/config.json")));
     CREATE_TABLE_WITHOUT_CURSOR_TYPE_QUERY = "CREATE TABLE %s (%s GEOMETRY)";
     INSERT_TABLE_WITHOUT_CURSOR_TYPE_QUERY = "INSERT INTO %s VALUES(ST_Point(129.77099609375, 62.093299865722656))";
   }
 
-  @BeforeEach
-  public void setup() throws Exception {
-    config = getStaticConfig();
-    super.setup();
+  @Override
+  protected RedshiftTestDatabase createTestDatabase() {
+    final RedshiftTestDatabase testDatabase = new RedshiftTestDatabase(source().toDatabaseConfig(Jsons.clone(config))).initialized();
+    try {
+      for (final String schemaName : TEST_SCHEMAS) {
```

Comment:
That's an interesting approach. I don't think that would work if there were enough activity on this connector that several CI would run concurrently, but since that's not the case, I think it's totally OK to push this change